### ISSUE TITLE
Net Plugin missing data of interface

### ISSUE THREAD

**Issue by ruudboon** [influxdata/telegraf#438]
I'm running Telegraf under FreeBSD 10.1. 
When using the net plugin the capturing of the lo0 and tun0 interface works but I'm unable to measure my main interface vtnet0. I specified the interface in the config but no data of this interface is showing when running.
```./telegraf -config telegraf.conf -test```
When I remove the lo0 and tun0 from the config no data for the net plugin show up at all.

**Comment by sparrc** [comment]
can you run `/usr/bin/netstat -ibdn` on your system and include the output?

**Comment by ruudboon** [comment]
Sure.

```
Name    Mtu Network       Address              Ipkts Ierrs Idrop     Ibytes    Opkts Oerrs     Obytes  Coll  Drop
vtnet  1500 <Link#1>      xx:xx:xx:xx:xx: 181815649     0     0 45944935428 77832474     0 20818708946     0     0 
vtnet     - xxxx::xxxx:xx xxxx::xxxx:xx:xxx        0     -     -          0        5     -        376     -     - 
vtnet     - xxx.xxx.xxx.x xxx.xxx.xxx.xx   52021498     -     - 12031916282 77795720     - 19728003532     -     - 
vtnet     - xxxx:xxx:xxxx xxxx::xxxx:xx:xxx        0     -     -          0        0     -          0     -     - 
lo0   16384 <Link#2>                        14056376     0     0 1354356495 14056388     0 1354352254     0     0 
lo0       - ::1/128       ::1                3381084     -     -  189341432  3381084     -  189341432     -     - 
lo0       - fe80::1%lo0/6 fe80::1%lo0              0     -     -          0        0     -          0     -     - 
lo0       - 127.0.0.0/8   127.0.0.1         10489923     -     - 1081860056 10491029     - 1081975401     -     - 
lo0       - xxx.xx.xxx.x/ xxx.xx.xxx.x        122136     -     -   27828924   121733     -   27241346     -     - 
lo0       - xxx.xx.xxx.x/ xxx.xx.xxx.x          7017     -     -    1862730     6615     -    1275211     -     - 
lo1   16384 <Link#3>                        219039113     0     0 68861815359 219040344     0 68862162252     0     0 
lo1       - xxx.xx.xxx.x/ xxx.xx.xxx.x      18995250     -     - 2126137867   126146     -   12816989     -     - 
lo1       - xxx.xx.xxx.x/ xxx.xx.xxx.x       1864667     -     -  699549108  1788162     -  674541837     -     - 
tun0   1500 <Link#4>                        23836315     0     0 2615696288 25065276     0 11588444658     0   161 
tun0      - xxxx::xxxx:xx xxxx::xxxx:xxxx:x        0     -     -          0        1     -        116     -     - 
tun0      - xx.x.xxx.x/32 xx.x.xxx.x         4485769     -     -  446534108 25065150     - 11488170207     -     - 
```

**Comment by ruudboon** [comment]
Founded it, vnet (without 0) is working. Kinda weird, because everywhere else in the sytem it called vtnet1

**Comment by sparrc** [comment]
thanks for tracking that down @ruudboon, I've pushed a PR to gopsutil to fix it there: https://github.com/shirou/gopsutil/pull/130

I'll close this case when it gets pulled into telegraf

**Comment by sparrc** [comment]
@ruudboon I noticed this in the man page:

```
-W    In certain displays, avoid truncating addresses even if this causes some fields
           to overflow.
```

Do you happen to know what happens when it overflows? I ask because gopsutil relies on there being a whitespace separation between the `Name` and `Mtu` fields.

**Comment by ruudboon** [comment]
@sparrc From what I see it keeps a white space between the columns. When scaling my terminal I do see line overflowing to the next line. But this is not different then without -W. It just happens sooner (in my case 1 character).
Thnx for making the PR

**Comment by sparrc** [comment]
closed by shirou/gopsutil#130
